cmake/modules/SwiftSharedCMakeConfig.cmake

```diff
@@ -264,6 +264,13 @@ macro(swift_common_unified_build_config product)
   endif()
 endmacro()
 
+# Common cmake project config for compilers.
+macro(swift_common_cxx_compiler_setup)
+  if("${CMAKE_C_COMPILER_ID}" STREQUAL "MSVC" OR "${CMAKE_CXX_SIMULATE_ID}" STREQUAL "MSVC")
+    set(SWIFT_COMPILER_IS_MSVC_LIKE TRUE)
+  endif()
+endmacro()
```

Comment:
I mean I don't see the point of having the macro to define the value.  I understand the value of having the MSVC_LIKE.